src/Pages/EndScreens/ScoreScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import { FunctionComponent } from "react";
+import { DefaultLayout } from "Layouts/DefaultLayout";
+import { navigate, RouteComponentProps } from "@reach/router";
+import { ScoreListType } from "types";
+import {
+  getDataFromLocalStorage,
+  StorageKeys,
+} from "services/getDataFromLocalStorage";
+import { RoutPaths } from "Pages/routes";
+
+export const ScoreScreen: FunctionComponent<RouteComponentProps> = () => {
+  const userScores = getDataFromLocalStorage<ScoreListType>(
+    StorageKeys.Scores,
+    []
+  );
+
+  return (
+    <DefaultLayout>
+      <div className="flex column align-center">
+        <h1>Your scores:</h1>
```

Comment:
on the page always should be only one h1 tag

Reply:
fixed 
now on the page exists only one h1; the second h1 was changed to h2